pennylane/pauli/utils.py

```diff
@@ -1494,6 +1531,11 @@ def _binary_matrix(terms, num_qubits, wire_map=None):
            [1, 0, 1, 0, 0, 0, 1, 0],
            [0, 0, 0, 1, 1, 0, 0, 1]])
     """
+    warn(
+        "_binary_matrix is deprecated. Instead, please use PauliWords and _binary_matrix_from_pws",
```

Comment:
Af far as I can see `_binary_matrix_from_pws`is also not used anywhere. Shouldn't that be deprecated as well? The same functionality is available publicly with `observables_to_binary_matrix`.

Reply:
it is used in various places in [qchem/tapering.py](https://github.com/PennyLaneAI/pennylane/blob/e643931ec29c6526cbcb98c10e59f3d6e863b887/pennylane/qchem/tapering.py#L28)